session/bootstrap.go

```diff
@@ -629,6 +635,10 @@ func upgradeToVer21(s Session) {
 	doReentrantDDL(s, "ALTER TABLE mysql.gc_delete_range DROP INDEX element_id", ddl.ErrCantDropFieldOrKey)
 }
 
+func upgradeToVer22(s Session) {
+	doReentrantDDL(s, "ALTER TABLE mysql.stats_histograms ADD COLUMN `stats_ver` bigint(64) NOT NULL DEFAULT 0", infoschema.ErrColumnExists)
```

Comment:
The `stats_ver` will be the last column. But it is not the last column in the create table statement. Please make it consistent.